js/popupGallery.js

```diff
@@ -0,0 +1,74 @@
+import {picturesContainer} from './renderingPicture.js';
+
+const displayingWindow = document.querySelector('.big-picture');
+const closeBigPhoto = displayingWindow.querySelector('.big-picture__cancel');
+const bigPicture = displayingWindow.querySelector('.big-picture__img img');
+const likesCounter = displayingWindow.querySelector('.likes-count');
+const commentCount = displayingWindow.querySelector('.comments-count');
+const commentsList = displayingWindow.querySelectorAll('.social__comments li');
+const descriptionPhoto = displayingWindow.querySelector('.social__caption');
+
```

Comment:
Обозначим "глобальную" переменную этого модуля с актуальным набором данных фотографий.
```suggestion

let photos = [];

```

Reply:
+